gems/rhino_project_core/app/controllers/concerns/rhino/error_handling.rb

```diff
@@ -23,14 +23,16 @@ module ErrorHandling
 
       rescue_from ActiveRecord::RecordNotFound, with: :not_found
 
-      def handle_uncaught_error(exception)
+      def handle_uncaught_error(e)
+        raise e if Rails.env.test?
```

Comment:
@silaspedrosa this seemed cleaner as all the back trace filtering still applies afaict